pcdsdevices/ipm.py

```diff
@@ -116,11 +133,11 @@ def remove(self, moved_cb=None, timeout=None, wait=False):
 
     @property
     def transmission(self):
-        """Returns the target's transmission value."""
-        return self.target.transmission
+        """Returns the combined transmission value of the target and diode"""
+        return self.target.transmission * self.diode.transmission
 
```

Comment:
This chooses 0.2 for unknown, it could be brought down to 0 if desired.
In and out are both full transmission.

Reply:
I think I prefer 0 under the assumption that we are not going to use the transmission of this calculation as safety device, but only as a notification that something is (might be..) in the beam. This can be changed later.